acceptance_tests/features/steps/events_emitted.py

```diff
@@ -181,3 +181,10 @@ def case_updated_emitted_with_correct_sensitve_data(context):
                             f'The emitted case, {emitted_case} does not match the case {context.case_id}')
     test_helper.assertEqual(emitted_case["sampleSensitive"]['PHONE_NUMBER'], "REDACTED",
                             "Expected emitted_case['sampleSensitive']['PHONE_NUMBER'] to equal 'REDACTED'")
+
+
+@step("A UAC_UPDATE event is sent for the record in the sample file")
+def step_impl(context):
+    emitted_uac = get_emitted_uac_update(context.correlation_id, context.originating_user)
```

Comment:
I would prefer this function to be more generic too, handle multiple rows in a file.  
there's 
"Then UAC_UPDATE messages are emitted with active set to "
this is used in multiple places.  You may be able to just use this function instead